src/components/Banner/Banner.tsx

```diff
@@ -60,120 +110,105 @@ export type BannerProps = {
  * ```tsx
  * <Banner
  *   onDismiss={handleDismiss}
- *   textContent={
+ *   text={
  *     <>
- *       <Banner.Title>{bannerTitle}</Banner.Title>
- *       <Banner.Message>{bannerMessage}</Banner.Message>
+ *       <BannerTitle>Some title</Banner.Title>
+ *       <BannerDescription>Some description</Banner.Description>
```

Comment:
should we update the example?

```
<Banner
    onDismiss={handleDismiss}
    title="Some title"
    description="Some description"
/>
```

Reply:
Yes, thank you! Also updated on `next`: https://github.com/chanzuckerberg/edu-design-system/pull/1021